glyphmaps/FontAwesome5Free.json

```diff
@@ -8,6 +8,7 @@
   "adn": 61808,
   "adversal": 62314,
   "affiliatetheme": 62315,
+  "air-freshener": 62928,
```

Comment:
Why did we suddenly get new free icons?

Reply:
Hello! I've been following this and felt like chiming in. :) Seems [this icon](https://fontawesome.com/icons/air-freshener?style=solid) came from the 5.2 release that [came out this week](https://fontawesome.com/changelog/latest).

Reply:
Yea, when I worked on the metadata generator it automatically updated the fonts so I just put them in there as well 🙂